04.ls/ls.rb

```diff
@@ -1,13 +1,18 @@
 # frozen_string_literal: true
 
+require 'optparse'
 COLUMN_COUNT = 3
 SPACE = 1
-
 def main
+  got_options = {}
```

Comment:
```suggestion
  options = {}
```

gotはあっても特に情報が増えないのでなくしていいと思います。